w06-ruby-on-rails/d3-dawn-ruby-oop/README.md

```diff
@@ -0,0 +1,176 @@
+#Ruby <3's OOP
+
+##Learning Objectives
+By the end of lesson, students will be able to...
+
+* Distinguish between objects & classes in Ruby
+* Initialize classes
+* Define attributes & instance methods
+* Distinguish between instance and class variables
+
+##Hashes (10m)
+
+Let's create a new Hash
+
+* Hashes are simple key/value stores.
+
+**Challenge:**
+How can I organize my data using key/value pairs in Ruby? Like so:
```

Comment:
I added an example of using an array and a hash as keys. Not a huge fan of discussing that just because it is not a great way to do things, but I guess they should know it is possible.